### DISCUSSION TITLE
过期：懒人包 2022EX

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#216] [回收]
## 版本专帖

_mpv已于近日启用官方Windows版构建（可用功能模块较shinchiro版缺失较多，应仅用于快速debug），参见其仓库的 [Actions](https://github.com/mpv-player/mpv/actions?query=branch%3Amaster) 板块。_

### 懒人包
今年最后一期整合。

### 简易包
无更新，沿用上期。

<hr />
链接 <a href='https://github.com/hooke007/MPV_lazy/releases/tag/20221112'>mpv-lazy-20221112</a>

链接 <a href='https://github.com/hooke007/mpv.net_CM/releases/tag/6032'>mpvnet-CM-20221014</a>

**Comment by wylsgame** [comment]
感谢分享！
下载了hook着色器补全，一共181个文件，都需要手动配置到mpv.conf吗？

**Reply by hooke007** [reply]
？？？？按需使用，用不到就不用下。而且全部塞进mpv.conf又是什么操作

**Reply by hlwyule** [reply]
为什么我下载解压后，双击mpv没有反应呀？

**Reply by wylsgame** [reply]
> ？？？？按需使用，用不到就不用下。而且全部塞进mpv.conf又是什么操作

想都试试看 看哪个效果好 @hooke007

**Reply by hooke007** [reply]
> 为什么我下载解压后，双击mpv没有反应呀？

如果是硬件老旧问题，需要手动替换掉自带的v3版mpv

**Comment by wylsgame** [comment]
请问如何让播放器标题栏显示filename 而不是title？

**Reply by hooke007** [reply]
播放器标题栏显示的不就是filename???

**Reply by wylsgame** [reply]
<img width="184" alt="image" src="https://user-images.githubusercontent.com/29670275/201911473-29bbfebc-45af-4d3b-a27a-ce35ceffb292.png">

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/discussions/69

**Comment by hooke007** [comment]
@s3327445

不要在那个线程里询问无关问题

1. trt对应rufe_cuda，等待cmd窗口生成引擎文件 https://github.com/hooke007/MPV_lazy/discussions/230
2. 要说差异也应该是移植的差异，没有人去动参数
3. 这不是锐化着色器

**Comment by hooke007** [comment]
- 无法复现的字体渲染怪癖 https://github.com/hooke007/MPV_lazy/discussions/245 https://github.com/hooke007/MPV_lazy/discussions/244

**Comment by taitaihoshi** [comment]
已经vo=gup-next、gpu-context=winvk了，增加 target-colorspace-hint=yes 不能实现hdr直通（include=profile那行已经#掉了），系统开启hdr开关之后，色调看起来像是映射在bt2020色域，但是在你最新版的mpvnet里倒是可以实现hdr直通，而且是能正确地映射到p3色域。

**Reply by hooke007** [reply]
没有条件复现

**Reply by taitaihoshi** [reply]
好吧，感谢回复，我试试把mpv.conf清空之后，只留下vo=gup-next，gpu-context=winvk，target-colorspace-hint=yes 试试能不能直通

**Reply by hooke007** [reply]
gup？

**Comment by wathper** [comment]
### 新版本的参数 `tscale = linear` 出现bug

系统配置：win10-64、RTX 30 系显卡、Intel CPU

将版本 **mpv-lazy-20220515** 升级到版本 **mpv-lazy-20221112** 后，新启用的参数 `tscale = linear` 在播放部分视频片段时会出现画面闪烁，疑似帧闪烁。

将 `tscale = linear` 修改为 `tscale = oversample` 或 `tscale = spline16` 则恢复正常，这是什么原因呢？

### 其次

新版本 **mpv-lazy-20221112** 中，在 **mpv.conf** 文件里新加入的滤镜 `glsl-shaders-append = "~~/shaders/AMD_FSR_EASU_rgb.glsl` 有什么作用呢？

它疑似作用于 AMD 显卡， RTX 显卡应该启用它吗？

### 最后

仓库中 **mpv-lazy-20221112-rifeCuda** 这个扩展包里具体有什么内容呢？
它解压后足有 2.5G 的大小，全部都是 RTX 显卡的补帧脚本吗？

另外，解压标准版 **mpv-lazy-20221112.exe** 后，我看到 **portable_confi** 文件夹中 **VS** 文件夹里已经配置好了作用于  RTX 显卡的补帧脚本 **"rife_cuda.vpy"** ，
它似乎和 **mpv-lazy-20221112-rifeCuda** 扩展包中 **VS** 文件夹里的 **"rife_cuda.vpy"** 脚本重复了，
我想使用 RTX 显卡专属补帧方案的话，还需要再额外安装 **mpv-lazy-20221112-rifeCuda** 扩展包吗？

**Reply by hooke007** [reply]
tscale=linear 无法复现

FSR 是类似传统scale的通用算法，与显卡无关

完整看完rife补帧帖

**Reply by wathper** [reply]
**对于 `tscale=linear`** 参数

直接解压使用 mpv-lazy-20221112 版不做修改，在我的机器上几乎播放所有视频都会触发该 bug ，我想它在某种系统环境下（例如我的系统）不适用。

---

**对于 rife_cuda.vpy 脚本的配置**

1. 将 `\mpv-lazy-20221112-rifeCuda\mpv-lazy\vapoursynth64` 文件夹与 `\mpv-lazy-20221112\mpv-lazy\vapoursynth64` 文件夹合并。
2. 将 `\mpv-lazy-20221112-rifeCuda\mpv-lazy` 中的 `vsmlrt.py` 脚本复制到 `\mpv-lazy-20221112\mpv-lazy`之中
3. 运行  `vsmlrt.py` 脚本

我这样做是正确的吗？

**Reply by hooke007** [reply]
这个包给你的是运行这个滤镜的先决条件；
运行一个py脚本没有意义，mpv从没支持过python脚本
所谓“一键包”，就是你只需要解压覆盖即部署完毕，不要做任何多余的操作。
部署完毕后该怎么开启滤镜就怎么开启

**Reply by wathper** [reply]
明白了

**Comment by hooke007** [comment]
@ssxwcz 你那改法和需求不匹配，直接删掉那个profile或者禁用这个profile

**Reply by ssxwcz** [reply]
好的，谢谢，通用文档反馈那个我也删除了，不好意思

**Comment by hula123** [comment]
为什么用这个mpv-lazy-20221112.exe 版本的rife背帧就很卡，但是我用那个cm汉化版本的rife就不卡很流畅，我把那个cm的model和rife.dll 都替换过来 rife_2x.py也替换至lazy版本,开启rife倍帧还是很卡 会丢帧，还有之所以这么做是感觉那个mpvnet_cm好像画质不容这个lazy版本好

**Reply by hooke007** [reply]
cm没什么预设参数

**Reply by hula123** [reply]
这个lazy版预设有哪些？glsl-shaders-append = "~~/shaders/AMD_FSR_EASU_rgb.glsl我看到前面有套论这个, 还有根据码率去色带的 别的还有啥？比较占资源的

**Reply by hooke007** [reply]
看着设置文件编辑呗，现在也没几个选项涉及性能的
